src/main/Main.ts

```diff
@@ -42,7 +45,7 @@ app.on("ready", () => {
     app.on("open-file", (event, file) => browserWindow.webContents.send("change-working-directory", file));
 });
 
-app.on("mainWindow-all-closed", () => process.platform === "darwin" || app.quit());
+app.on("window-all-closed", app.quit );
```

Comment:
yeah sorry. My commits are so strange... I can't really work on one topic at a time. But yes. For now simply ignore it, I will make the Black Screen work in multi windows, but for now it would just fix the crash when you close a window but your menu is still there and does not function...